cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -206,6 +214,13 @@ is_precompiled_driver() {
   [[ "${INSTALLER_FILE##*.}" == "cos" ]] && has_precompiled_driver_signature || return $?
 }
 
+# Download kernel_info file
+download_kernel_info_file() {
+  local -r kernel_info_file_path="${1}/${COS_KERNEL_INFO_FILENAME}"
```

Comment:
Can you please assign ${1} to an variable first before using it? It's not clear what is ${1}.

e.g.

local -r cos_build_gcs_path=${1}
 local -r kernel_info_file_path="${cos_build_gcs_path}/${COS_KERNEL_INFO_FILENAME}"

Reply:
Done.